core/src/main/java/org/wso2/testgrid/core/command/RunTestPlanCommand.java

```diff
@@ -127,7 +132,8 @@ public void execute() throws CommandExecutionException {
                     testConfig.getDeploymentPatterns().get(0));
 
             // Generate test plan from config
-            TestPlan testPlan = generateTestPlan(deploymentPattern, testConfig, scenarioRepoDir, infraRepo);
+            TestPlan testPlan = generateTestPlan(deploymentPattern, testConfig, scenarioRepoDir, infraRepo,
+                    deploymentRepo);
```

Comment:
This is not an issue in this code - but, the API in #generateTestPlan's last three input parameter order is inconsistent. It should be infraRepo, deploymentRepo, and scenarioRepo respectively.